Project Requirements Document.md

```diff
@@ -0,0 +1,27 @@
+|            **Project Name**            | Threewide                                                                                                                                                                                                    |
+|       **Functional Requirements**      |                                                                                                                                                                                                              |
+|            **High Priority**           | 1. Beginners can play Tetris game with modern tetris ruleset.                                                                                                                                                |
+|                                        | 2. Beginners can learn strategies to apply into game.                                                                                                                                                            |
+|                                        | 3. Our primary goal is to teach T spin by asking the user to perform n amount of them in small gmaes embedded on Threewide. It also includes teaching of Super Rotation System Kick Table, DAS and ARP etc.                                                                                                                                          |
+|                                        | 4. Threewide should be able to verify that the user accomplished that goal.                                                                                                                                         |
+|            **Low Priority**            | 1. Add more customization to the Tetris game on our plateform for beginners. For instance, we can add teaching strategies for other Tetris blocks such as Z,J,L,S tetrominoes etc. |
+|                                        | 2. Add some advance learning material on higher level.                                                                                                                                                       |
+| **Technical/Performance Requirements** | 1. React.js - React.js to design attractive and interactive interface for the beginners.                                                                                                                     |
+|                                        | 2. Node.js - Node.js will be used to render the pages of our platform ThreeWide. Node.js will be used to provide an embedded game and a React application with Tetris problem data to the user.                                                 |
+|                                        |    3. MongoDB - MongoDB database will be used to store and managed essential data about users and their progress level on Threewide. |
+
+
+
```

Comment:
Nitpick: can you remove all these new lines

Reply:
Fixed